operators/endpointmetrics/manifests/prometheus/prometheus-role-default.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  annotations:
+    update-namespace: "false"
+  name: prometheus-k8s
+  namespace: default
```

Comment:
Also this namespace `default`, is it intent?

Reply:
@morvencao 
both namespace "default" is desired.
it's from https://github.com/prometheus-operator/kube-prometheus/blob/996d5163639371925e101d24e36406871991b8b1/manifests/prometheus-roleSpecificNamespaces.yaml#L12

Reply:
OK